frontend/src/tutorial/Tutorial.tsx

```diff
@@ -58,16 +54,14 @@ const Tutorial: React.FC<TutorialProps> = ({ name, steps, zIndex }) => {
                         zIndex: zIndex || 100,
                     },
                 }}
-                disableCloseOnEsc
-                disableOverlayClose
```

Comment:
Can we leave disableOverlayClose that was somebody doesn't accidentally close it?

Reply:
The first time I used it I kept clicking on the overlay trying to get it to close. It's the trained behavior of many apps that clicking away from the dialog closes it. Maybe this is a question for Post?